ColorPatternsTests/ColorPatternsTests.swift

```diff
@@ -8,8 +8,12 @@
 
 import XCTest
 
+@testable import ColorPatterns
+
 class ColorPatternsTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/de3fc83bd5fbc95a2862a6bf4bc68d568b18dd99/ColorPatternsTests/ColorPatternsTests.swift#L13"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>